render-app/src/main/java/org/janelia/alignment/match/OrderedCanvasIdPair.java

```diff
@@ -55,6 +57,47 @@ public OrderedCanvasIdPair(final CanvasId oneId,
         this.absoluteDeltaZ = deltaZ == null ? null : Math.abs(deltaZ);
     }
 
+    /**
+     * @param  oneTileBounds      identifiers and bounds for one canvas identifier.
+     * @param  anotherTileBounds  identifiers and bounds for another canvas identifier.
+     *
+     * @return an ordered pair where each CanvasId includes relative position information
+     *         based upon the specified tile bounds.
+     *         Assumes that the bounds are for the same layer so the pair's deltaZ is set to null.
+     *
+     * @throws IllegalArgumentException
+     *   if both tile identifiers are the same.
+     */
+    public static OrderedCanvasIdPair withRelativePositions(final TileBounds oneTileBounds,
+                                                            final TileBounds anotherTileBounds)
+            throws IllegalArgumentException {
+
+        final CanvasId oneCanvasId = new CanvasId(oneTileBounds.getSectionId(), oneTileBounds.getTileId());
+        final CanvasId anotherCanvasId = new CanvasId(anotherTileBounds.getSectionId(), anotherTileBounds.getTileId());
+
+        final MontageRelativePosition[] positions = MontageRelativePosition.getRelativePositions(oneTileBounds,
```

Comment:
It seems to me that the signature of `MontageRelativePosition.getRelativePosition` (which I would rename to `MontageRelativePosition.of` and only let it return one position instead of an array) is pretty much tailored to this use here and that some of the responsibility (e.g., checking if two `TileBounds` are the same -- maybe introduce `MontageRelativePosition.SAME`?) depend on the implementation within `MontageRelativePosition` and therefore should go there.